### ISSUE TITLE
Error running example

### ISSUE THREAD

**Issue by HyperManTT** [douglasrizzo/catsim#14]
Hey,

Not sure if this project is still supported but I'm trying to run the "Autonomous Usage" code that was shown in the docs and I'm getting the below error. Can you say if something has changed or needs to be updated?

```
File "/Users/rram/PycharmProjects/CAT_Testing/mac_venv/lib/python3.7/site-packages/catsim/selection.py", line 83, in <listcomp>

valid_indexes_low_r = [index for index in valid_indexes if items[index, 4] < self._r_max]

IndexError: index 4 is out of bounds for axis 1 with size 4```

**Comment by douglasrizzo** [comment]
Hi there @HyperManTT , thanks for showing interest in the project. While I've not been implementing anything new on catsim, since I'm not working with adaptive tests anymore, I am still interested in fixing bugs in the project and keeping it usable. I will take a look at this issue in the coming days.

**Comment by HyperManTT** [comment]
Thanks @douglasrizzo. Thanks for the response. Really appreciate it.

**Comment by douglasrizzo** [comment]
I found the problem. When the `Simulator` object receives the item matrix, if this matrix doesn't have 5 columns (4 for the item parameters and the last one for the exposure rate), the simulator adds the last column before starting simulations. You can see it in [these lines](https://github.com/douglasrizzo/catsim/blob/2b5f2b95cee68171d22e6056bf4add14ec6db435/catsim/simulation.py#L145-L147).

However, when the components are used autonomously (outside of a `Simulator` object) this column is not created and we get an *out of bounds* error. The correct thing to do, I believe, would be to check which components mention this column and which ones of these can either generate the column, ignore its value or throw a more informative error. I believe only `selector`s actually mention that column, but I'll have to check that.

**Comment by douglasrizzo** [comment]
I have pushed a few commits to the testing branch. I have tested the "autonomous usage" examples and they are working now. As soon as the automated tests are finished, I'll push a new version do PyPI and to the master branch.

**Comment by douglasrizzo** [comment]
This has been fixed in d6259796d00d139cfbeea8cc337cf431801e95c0.
